app/concepts/base/repository.rb

```diff
@@ -0,0 +1,42 @@
+module Repositories
+    class Base 
```

Comment:
I think this class should name `Repository` and be within the `Base` module.

Reply:
@baniak66 Ok, sounds fine, I'm gonna rename it.

Reply:
Well I'm gonna rename it eventually, so autoloader zeitwork works, now It loads classic.
https://medium.com/cedarcode/understanding-zeitwerk-in-rails-6-f168a9f09a1f